doc/updating_documentation/How_to_update_this_documentation.md

```diff
@@ -0,0 +1,28 @@
+# How to update this documentation
+
+## How does it work
+
+Documentation is stored on master branch of Juniper/contrail-windows-docs repository. It is stored in Markdown (.md) format. Documentation is served as HTML on github.io. HTML pages are generated by MkDocs tool. HTML pages need to be redeployed manually after modification.
+
+## Documentation update procedure (procedure for developer)
+
+1. Open a pull request to master branch in repository Juniper/contrail-windows-docs with your changes.
+2. Once PR is accepted, an administrator is going to deploy new modifications to gh-pages branch.
+
+## How to preview modifications (procedure for developer)
+
+1. Install MkDocs (see below).
+2. Apply required modifications to the documentation.
+3. Invoke `mkdocs serve` in main repository directory. It's going to start local HTTP server with documentation preview. Follow instructions displayed by mkdocs to see the outcome.
+
+## How to install MkDocs (procedure for administrator and developer)
+
+MkDocs requires Python (both Python 2 and Python 3 are supported) and PIP. Follow instructions on [http://www.mkdocs.org/#installation](http://www.mkdocs.org/#installation). Basically `pip install mkdocs` should work. It may also be useful to use virtualenv to keep mkdocs and its dependencies in one isolated place.
```

Comment:
Maybe we should provide a requirements file with locked `mkdocs` version (and its dependencies)?